### PR TITLE
refactor: Simplify deployment info for LSP16 & LSP23

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

This PR simplifies the information related to deploying LSP16 & LSP23 at vanity addresses.

### CODE REVIEW

**Review by skimaharvey — APPROVED** [review]

**Review by YamenMerhi — APPROVED** [review]

File: LSPs/LSP-16-UniversalFactory.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ created: 2022-11-19
 requires: EIP104, EIP155, EIP1167
 ---
 
+**The official LSP16 Universal Factory address on LUKSO is: [0x1600016e23e25D20CA8759338BfB8A8d11563C4e](https://explorer.lukso.network/address/0x1600016e23e25D20CA8759338BfB8A8d11563C4e).**
+
+To deploy this follower system on other chains please see the [deployment section](#deployment).
```
Comment (JeneaVranceanu) [inline LSPs/LSP-16-UniversalFactory.md:14]: Don't understand how follower system is related to LSP-16 and 23 in this context.
Nothing about "follower system" is mentioned or explained here or in LSP-23.

Comment (CJ42) [inline LSPs/LSP-16-UniversalFactory.md:14]:
```suggestion
```

File: LSPs/LSP-16-UniversalFactory.md [hunk 2]
```diff
@@ -743,6 +277,24 @@ Knowing that deploying a contract using the UniversalFactory will allow to deplo
 
 The constructor parameters or/and initialization data SHOULD NOT include any network-specific parameters (e.g: chain-id, a local token contract address), otherwise the deployed contract will not be recreated at the same address across different networks, thus defeating the purpose of the UniversalFactory.
 
+## Deployment
+
+The `LSP16UniversalFactory` is deplpoyed at [`0x1600016e23e25D20CA8759338BfB8A8d11563C4e`](https://explorer.lukso.network/address/0x1600016e23e25D20CA8759338BfB8A8d11563C4e) on LUKSO using the [Nick Factory contract](https://github.com/Arachnid/deterministic-deployment-proxy/tree/master). The following explains how to deploy `LSP16UniversalFactory` at the same address on other EVM networks.
```
Comment (CJ42) [inline LSPs/LSP-16-UniversalFactory.md:282]:
```suggestion
The `LSP16UniversalFactory` is deployed at [`0x1600016e23e25D20CA8759338BfB8A8d11563C4e`](https://explorer.lukso.network/address/0x1600016e23e25D20CA8759338BfB8A8d11563C4e) on LUKSO using the [Nick Factory contract](https://github.com/Arachnid/deterministic-deployment-proxy/tree/master). The following explains how to deploy `LSP16UniversalFactory` at the same address on other EVM networks.
```

File: LSPs/LSP-23-LinkedContractsFactory.md [hunk 3]
```diff
@@ -757,6 +343,24 @@ interface IPostDeploymentModule {
 }
 ```
 
+## Deployment
+
+The `LSP23LinkedContractsFactory` is deplpoyed at [`0x2300000A84D25dF63081feAa37ba6b62C4c89a30`](https://explorer.lukso.network/address/0x2300000A84D25dF63081feAa37ba6b62C4c89a30) on LUKSO using the [Nick Factory contract](https://github.com/Arachnid/deterministic-deployment-proxy/tree/master). The following explains how to deploy `LSP23LinkedContractsFactory` at the same address on other EVM networks.
```
Comment (CJ42) [inline LSPs/LSP-23-LinkedContractsFactory.md:348]:
```suggestion
The `LSP23LinkedContractsFactory` is deployed at [`0x2300000A84D25dF63081feAa37ba6b62C4c89a30`](https://explorer.lukso.network/address/0x2300000A84D25dF63081feAa37ba6b62C4c89a30) on LUKSO using the [Nick Factory contract](https://github.com/Arachnid/deterministic-deployment-proxy/tree/master). The following explains how to deploy `LSP23LinkedContractsFactory` at the same address on other EVM networks.
```

**Review by CJ42 — CHANGES_REQUESTED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**